extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -18,19 +18,17 @@ syntax = "proto3";
 package stackdriver.config.v1alpha1;
 
 message PluginConfig {
-  // Indicates whether the module should act as inbound or outbound reporter.
-  // Difference being:
-  //   * inbound and outbound have different metric types to export.
-  //   * inbound reporter writes access logs.
-  enum ReporterKind {
-    INBOUND = 0;
-    OUTBOUND = 1;
-  }
-  ReporterKind kind = 1;
-
   // Controls whether to export access log.
-  bool disable_access_logging = 2;
+  bool disable_access_logging = 1;
 
   // FQDN of destination service that the request routed to.
-  string destination_service_name = 3;
+  string destination_service_name = 2;
```

Comment:
can we provide an example here?  and maybe clarify/distinguish between `destination_service_name` and `destination_service_host`?

Reply:
Also, is this required?  what is the implication of this information not being populated?

Reply:
No it is not required and it suppose to be filled by control plane. If not provided, request host will be used instead.